bin/sentieon-genomics-202010.02/doc/licsrvr.sh

```
@@ -0,0 +1,70 @@
+licfile=${licfile:-/etc/sentieon.lic}
+logfile=${logfile:-/var/log/licsrvr.log}
+
+start() {
+	echo -n "Starting $prog:"
+	$licsrvr --log $logfile --start $licfile
+	RETVAL=$?
+	echo
+}
+
+stop() {
+	echo -n "Stopping $prog:"
```

Shellcheck (0.10.0)

[warning] In POSIX sh, echo flags are undefined.

(SC3037)